src/routes/email/guards/timestamp.guard.ts

```diff
@@ -13,7 +13,9 @@ export const TimestampGuard = (maxElapsedTimeMs: number): Type<CanActivate> => {
     canActivate(context: ExecutionContext): boolean {
       const request = context.switchToHttp().getRequest();
 
-      const timestampRaw = request.body['timestamp'];
+      const timestampRaw =
+        request.body['timestamp'] ??
+        request.headers['safe-wallet-signature-timestamp'];
```

Comment:
Had to add this fallback since this guard was expecting the timestamp to be given in the body of the request.